contracts/modules/cover/Cover.sol

```diff
@@ -109,6 +111,12 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
     globalCapacityRatio = 20000; // x2
     globalRewardsRatio = 5000; // 50%
     coverAssetsFallback = 3; // 0x11 - DAI and ETH
+    uint64 bucketIdStart = (block.timestamp / BUCKET_SIZE).toUint64();
+    uint assetId;
+    while (assetId <= coverAssetsFallback) {
+      activeCover[assetId].lastBucketUpdateId = bucketIdStart;
+      ++assetId;
+    }
```

Comment:
Each `assetId` should have a single bit `1` and others `0` (ex `0b010`). Current loop will go through all numbers instead of just looping through valid asset ids.